x-pack/plugins/ml/server/routes/schemas/alerting_schema.ts

```diff
@@ -58,3 +60,47 @@ export type MlAnomalyDetectionAlertParams = TypeOf<typeof mlAnomalyDetectionAler
 export type MlAnomalyDetectionAlertPreviewRequest = TypeOf<
   typeof mlAnomalyDetectionAlertPreviewRequest
 >;
+
+export const anomalyDetectionJobsHealthRuleParams = schema.object({
+  includeJobs: jobsSelectionSchema,
+  excludeJobs: schema.nullable(jobsSelectionSchema),
```

Comment:
Is this an error because it expects a value for 'exclude jobs'? If it's a valid error, is there anything we can do to make the text more user-friendly.

![image](https://user-images.githubusercontent.com/7405507/126321825-beac3ed2-69a6-4b51-9b04-d5c0b92402fa.png)

Reply:
No, apparently it happed because you added something to excluded jobs and then deleted it, and it passed on empty object eventually, but Kibana schema validation didn't like it. thinking how to fix it

Reply:
Fixed in https://github.com/elastic/kibana/pull/106084/commits/b552dda411d30cc75523b2b4b1df62d801ae0c59